tests/scripts/task_python_vta.sh

```diff
@@ -19,35 +19,45 @@
 set -e
 set -u
 
```

Comment:
why do we need to change this? looks it is vta related stuff here

Reply:
The i386 instances in the CI use a 32-bit version of Python which isn't able to install PyTorch so I added changes to skip the tests based on the Python version for each script which had a conflict (basically anything that imports TVM frontends). I talked with Philip a little bit regarding this issue but I'm open to other approaches.

Reply:
not sure if it is possible to create a new file like "tests/scripts/task_python_neopytorch.sh" to do all the required settting?  cc some folks: @hcho3 @zhiics @kevinthesun @wweic

Reply:
Some options are to update i386 machine to use 64bit python like the others (not sure if possible) or remove load_model and import torch in the parser and take in a trace (which is what it originally is doing and probably what we will do in the upstream PR). This way, the import statement is in the frontend/.../pytorch/test_forward.py which only runs on GPU anyway so other tests are not affected. This will require adding dependency to the internal NeoCompiler service code to handle the tracing though.

Reply:
Going to go with the latter option (take in trace and make changes to service code to accommodate).